bndtools.core/src/bndtools/central/Central.java

```diff
@@ -461,6 +465,9 @@ public static void refreshPlugins() throws Exception {
                         l.modelChanged(p);
                 }
 
+                if (refreshedRepos.size() > 0) {
+                    repositoriesViewRefresher.repositoriesRefreshed();
```

Comment:
I am talking about this method:

```java
    public static void refreshPlugin(Refreshable plugin) throws Exception {
        if (plugin.refresh()) {
            refreshFile(plugin.getRoot());
            for (Project p : Central.getWorkspace().getAllProjects()) {
                p.setChanged();
                for (ModelListener l : getInstance().listeners)
                    l.modelChanged(p);
            }
        }
    }
```

If it is called with a RepositoryPlugin argument, should the method not call `repositoryRefreshed(RepositoryPlugin)` on the argument?